src/core/control/jobs/ImageExport.h

```diff
@@ -46,49 +46,43 @@ class RasterImageQualityParameter {
      * @brief Get the quality criterion of this parameter
      * @return The quality criterion
      */
-    ExportQualityCriterion getQualityCriterion();
+    auto getQualityCriterion() -> ExportQualityCriterion const;
 
     /**
      * @brief Get the target value of this parameter
      * @return The target value
      */
-    int getValue();
+    auto getValue() -> int const;
 
 private:
     /**
      * @brief Default quality is: DPI=300
      */
-    ExportQualityCriterion qualityCriterion = EXPORT_QUALITY_DPI;
-    int value = 300;
+    ExportQualityCriterion qualityCriterion{EXPORT_QUALITY_DPI};
+    int value{300};
```

Comment:
I think the better readability of `=` initialization outweights to precision loss checks (that we get anyway, as compiler warnings). This seems to have been a consensus amongst the devs here (correct me if I'm wrong).
Ofc, this applies elsewhere in your code as well.

Reply:
Should be changed accordingly at all parts.